config/webpack/webpack.desktop.ts

```diff
@@ -1,29 +1,29 @@
-const path = require('path');
-const webpack = require('webpack');
-const _ = require('underscore');
-
-const desktopDependencies = require('../../desktop/package.json').dependencies;
-const getCommonConfig = require('./webpack.common');
+/* eslint-disable @typescript-eslint/naming-convention */
+import path from 'path';
+import type {Configuration} from 'webpack';
+import webpack from 'webpack';
+// eslint-disable-next-line @dword-design/import-alias/prefer-alias, import/no-relative-packages -- alias imports don't work for webpack
+import {dependencies as desktopDependencies} from '../../desktop/package.json';
+import type Env from './types';
+import getCommonConfig from './webpack.common';
 
 /**
  * Desktop creates 2 configurations in parallel
  * 1. electron-main - the core that serves the app content
  * 2. web - the app content that would be rendered in electron
  * Everything is placed in desktop/dist and ready for packaging
- * @param {Object} env
- * @returns {webpack.Configuration[]}
  */
-module.exports = (env) => {
+const getConfig = (env: Env): Configuration[] => {
     const rendererConfig = getCommonConfig({...env, platform: 'desktop'});
     const outputPath = path.resolve(__dirname, '../../desktop/dist');
 
     rendererConfig.name = 'renderer';
-    rendererConfig.output.path = path.join(outputPath, 'www');
+    (rendererConfig.output ??= {}).path = path.join(outputPath, 'www');
 
     // Expose react-native-config to desktop-main
```

Comment:
Is this comment above the line below it?

Reply:
I think it's above, not 100% sure what the author had in mind though

Reply:
🙈 I phrased that very poorly, but I think you got it. How does this line _expose react-native-config to desktop-main_? Isn't that done by `plugins: [definePlugin],`? this isn't a blocker

Reply:
By "author" I meant the person that added the comment 😄